update-ca-certificates: Replace deprecated mktemp -t with mktemp --tmpdir

According to coreutils docs, mktemp -t is deprecated, switch to the
--tmpdir option instead.

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
Upstream-Status: Submitted [https://salsa.debian.org/debian/ca-certificates/-/merge_requests/5]

[This was originally for compatibility with toybox but toybox now
supports -t]
---
 sbin/update-ca-certificates | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sbin/update-ca-certificates b/sbin/update-ca-certificates
index 79c41bb..ae9e3f1 100755
--- a/sbin/update-ca-certificates
+++ b/sbin/update-ca-certificates
@@ -113,9 +113,9 @@ trap cleanup 0
 
 # Helper files.  (Some of them are not simple arrays because we spawn
 # subshells later on.)
-TEMPBUNDLE="$(mktemp -t "${CERTBUNDLE}.tmp.XXXXXX")"
-ADDED="$(mktemp -t "ca-certificates.tmp.XXXXXX")"
-REMOVED="$(mktemp -t "ca-certificates.tmp.XXXXXX")"
+TEMPBUNDLE="$(mktemp --tmpdir "${CERTBUNDLE}.tmp.XXXXXX")"
+ADDED="$(mktemp --tmpdir "ca-certificates.tmp.XXXXXX")"
+REMOVED="$(mktemp --tmpdir "ca-certificates.tmp.XXXXXX")"
 
 # Adds a certificate to the list of trusted ones.  This includes a symlink
 # in /etc/ssl/certs to the certificate file and its inclusion into the
-- 
2.1.4
